docs/src/main/asciidoc/rest-client.adoc

```diff
@@ -191,8 +189,8 @@ The name of the property needs to follow a certain convention which is best disp
 [source,properties]
 ----
 # Your configuration properties
-org.acme.rest.client.CountriesService/mp-rest/url=https://restcountries.eu/rest # // <1>
-org.acme.rest.client.CountriesService/mp-rest/scope=javax.inject.Singleton # // <2>
+quarkus.rest-client."org.acme.rest.client.CountriesService".url=https://restcountries.eu/rest # // <1>
```

Comment:
I'm ok if we recommend using the quarkus properties instead, but the old ones are still valid, so I think we should add a note to clarify which ones have priority if you have both set (shouldn't happen, but still....)

Reply:
I added a short note stating that the "standard MicroProfile Rest Client notation" can be used too, while Quarkus notation takes a precedence. Thanks @radcortez!